manifests/init.pp

```diff
@@ -234,6 +240,8 @@
   Stdlib::Absolutepath                                $network_path = '/etc/systemd/network',
   Hash                                                $loginctl_users = {},
   Hash                                                $dropin_files = {},
+  Hash                                                $manage_units = {},
+  Hash                                                $manage_dropins = {},
```

Comment:
Others don't have it either, but shouldn't this be narrowed down to `Hash[String[1], Hash]`?

Reply:
Thinking more about it: almost feels like a `Stdlib::CreateResources` type would make sense.

Reply:
Also crossed my mind.

We'd have to bump minimum stdlib from 4.13.1 to 8.5.0 https://github.com/puppetlabs/puppetlabs-stdlib/commit/e8ba623bbd7facbad726e9c56b39a8c5341c16c8

Suggest follow up with a backward-incompatible MR that changes the 3(?) of them.

Reply:
Though I really wonder who would such an old stdlib, is this really breaking?

Reply:
Change the two new ones.
Will follow up with the rest and it can sit there for rainy day.

Reply:
> We'd have to bump minimum stdlib from 4.13.1 to 8.5.0 [puppetlabs/puppetlabs-stdlib@e8ba623](https://github.com/puppetlabs/puppetlabs-stdlib/commit/e8ba623bbd7facbad726e9c56b39a8c5341c16c8)

I'll admit: I completely forgot I already wrote that and was just theorizing "it would be nice", but then thought "it'll be a while before everyone has it". Looks like I already did the hard parts long enough ago.